ddtrace/_trace/tracer.py

```diff
@@ -227,7 +227,8 @@ def __init__(
                     category=DDTraceDeprecationWarning,
                     removal_version="3.0.0",
                 )
-        self._filters: List[TraceFilter] = []
+
+        self._user_trace_processors: List[TraceProcessor] = []
```

Comment:
since filter inherited from processor, this seems fine.

Reply:
Ya I want to get rid of the concept of TraceFilters in v4.0. We should use TraceProcessors everywhere